cmd/aries-agent-rest/go.mod

```diff
@@ -8,9 +8,10 @@ replace github.com/hyperledger/aries-framework-go => ../..
 
 require (
 	github.com/gorilla/mux v1.7.3
-	github.com/hyperledger/aries-framework-go v0.0.0
+	github.com/hyperledger/aries-framework-go v0.1.1
```

Comment:
should stay v0.0.0

Reply:
@rolsonquadras  Note the replace line which overrides the import: https://github.com/hyperledger/aries-framework-go/pull/1236/files#diff-72a26b100caddedd2c0dbe357cbc5b1fR7

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/b6921cdf5ecf204f9802ff1bca9887b2835ae673..38a85a8a82952cb9beefce98ee829c9ccf0276f6).